examples/cloud/query.ts

```diff
@@ -1,18 +1,32 @@
+import fs from "node:fs/promises";
+
 import { stdin as input, stdout as output } from "node:process";
+
 import readline from "node:readline/promises";
 
-import { LlamaCloudIndex } from "llamaindex";
+import { Document, LlamaCloudIndex } from "llamaindex";
 
 async function main() {
-  const index = new LlamaCloudIndex({
+  const path = "node_modules/llamaindex/examples/abramov.txt";
+
+  const essay = await fs.readFile(path, "utf-8");
+
+  // Create Document object with essay
+  const document = new Document({ text: essay, id_: "file_abramov.txt" });
+
+  const index = await LlamaCloudIndex.fromDocuments({
```

Comment:
the query example is just to query the pipeline which is generated with `pipeline.ts`. 
`.fromDocuments` is an alternative to the code in `pipeline.ts`, so I would move it there (are create a new file)